### DISCUSSION TITLE
Where is "gst" echo server image source code ?

### DISCUSSION THREAD

**Discussion by shinyoshiaki** [sipsorcery/webrtc-interop#27] [Q&A]
@sipsorcery 
I want to debug it in my local environment.
so If you can publish the source code, I want you to publish it.

**Comment by sipsorcery** [comment]
The code, build and docker files are available [here](https://github.com/sipsorcery/prototypes/tree/master/gst-webrtc-echo). It's still a bit of a work in progress. When it's in a bit better shape I'll add it to this repo.

**Reply by shinyoshiaki** [reply]
thanks !
